src/controller/python/chip/yaml/runner.py

```diff
@@ -266,6 +279,51 @@ def parse_raw_response(self, raw_resp) -> _ActionResult:
         return _ActionResult(status=_ActionStatus.SUCCESS, response=return_val)
 
 
+class ReadEventAction(BaseAction):
+    ''' Read Event action to be executed.'''
+
+    def __init__(self, test_step, cluster: str, context: _ExecutionContext):
+        '''Converts 'test_step' to read event action that can execute with ChipDeviceController.
+
+        Args:
+          'test_step': Step containing information required to run read event action.
+          'cluster': Name of cluster read event action is targeting.
+          'context': Contains test-wide common objects such as DataModelLookup instance.
+        Raises:
+          UnexpectedParsingError: Raised if there is an unexpected parsing error.
+        '''
+        super().__init__(test_step)
+        self._event_name = stringcase.pascalcase(test_step.event)
+        self._cluster = cluster
+        self._endpoint = test_step.endpoint
+        self._node_id = test_step.node_id
+        self._cluster_object = None
+        self._request_object = None
+        self._fabric_filtered = True
```

Comment:
I suspect that you want that do be defined per the `fabric_filtered` value if set.